### PR TITLE
Added v6.0 workaround for upgrade error

### PR DESCRIPTION
**Pull request by cwarnermm**
Updated: Deploy Mattermost > Upgrade Mattermost > Upgrading Mattermost Server > Upgrading from Releases Older than v5.35 to:
- add a note that details a workaround for a specific error that can be encountered during the upgrade

### CODE REVIEW

File: source/upgrade/upgrading-mattermost-server.rst [hunk 1]
```diff
@@ -87,7 +87,29 @@ If you're upgrading from a version prior to Mattermost v5.0, you can't upgrade d
...
 Ensure you review the :doc:`important-upgrade-notes` for all intermediate release versions in between to ensure you’re aware of the possible migrations that could affect your upgrade.
 
-Upgrading High Availability Deployments
+.. note::
+
+  Customers upgrading from releases older than v5.35 following our recommended upgrade process may encounter the following error during the upgrade to v6.0:
+  
+  ``Failed to alter column type. It is likely you have invalid JSON values in the column. Please fix the values manually and run the migration again.","caller":"sqlstore/store.go:854","error":"pq: unsupported Unicode escape sequence``
+  
+  This issue will be addressed in a patch release, but in the interim, you can enable ``SqlSettings.Trace`` to narrow down what table and column are causing issues during the upgrade: The following queries change the columns to JSONB format in PostgreSQL. Run these against your v5.39 development database to find out which table and column has Unicode issues:
```
[the remark this answers is not in the document]
Comment (wiersgallak) [reply source/upgrade/upgrading-mattermost-server.rst:96]: My recommendation is below: 
```suggestion
  To assist with troubleshooting, you can enable ``SqlSettings.Trace`` to narrow down what table and column are causing issues during the upgrade. The following queries change the columns to JSONB format in PostgreSQL. Run these against your v5.39 development database to find out which table and column has Unicode issues:
```

**Comment by cwarnermm** [comment]
@streamer45 @wiersgallak - I've applied all technical review updates and this PR should now be ready for final review and approval.

**Review by streamer45 — APPROVED** [review]
Looks good :+1:

**Review by wiersgallak — APPROVED** [review]
Thanks Carrie and team!

File: source/upgrade/upgrading-mattermost-server.rst [hunk 2]
```diff
@@ -87,7 +87,39 @@ If you're upgrading from a version prior to Mattermost v5.0, you can't upgrade d
 
 Ensure you review the :doc:`important-upgrade-notes` for all intermediate release versions in between to ensure you’re aware of the possible migrations that could affect your upgrade.
 
-Upgrading High Availability Deployments
+.. note::
+
+  Customers upgrading from releases older than v5.35 following our recommended upgrade process may encounter the following error during the upgrade to v6.0:
+  
+  `Failed to alter column type. It is likely you have invalid JSON values in the column. Please fix the values manually and run the migration again.","caller":"sqlstore/store.go:854","error":"pq: unsupported Unicode escape sequence`
```
Comment (cwarnermm) [inline source/upgrade/upgrading-mattermost-server.rst:94]:
```suggestion
  ``Failed to alter column type. It is likely you have invalid JSON values in the column. Please fix the values manually and run the migration again.","caller":"sqlstore/store.go:854","error":"pq: unsupported Unicode escape sequence``
```

**Pull request merged**